docs/content/docs/testing/telemetry.md

```diff
@@ -33,7 +33,7 @@ The `Loader` constructor is called by both `createContainer()` and `getContainer
 interface as its constructor argument. `ILoaderProps` interface has an optional logger parameter that will take the
 `ITelemetryBaseLogger` defined by the user.
 
-[ILoaderProps.logger](https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/src/loader.ts#L313)
+[ILoaderProps.logger](https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/src/loader.ts#L289)
```

Comment:
This is tracking a "moving target" by pointing to the code in the main branch instead of a fixed version... not sure how I feel about the options. Pointing to a fixed version could feel weird if someone gets here months from now after the file has changed significantly and gets taken to an earlier version.

Reply:
Yeah, this doesn't seem great. Maybe it would be better to point to the API docs (assuming the item being referenced is non-internal)?

Reply:
Or just point to the package and just reference the particular API name.

Reply:
Done. As I was fixing it, I also realized that it's v2 documentation pointing to legacy APIs so it probably shouldn't even be there in the first place. But I'll keep the focus of this PR on improving the links situation, and actual docs cleanup can be separate.